python/astro_metadata_translator/tests.py

```diff
@@ -144,15 +144,19 @@ def assertObservationInfoFromYaml(self, file, dir=None, check_wcs=True,  # noqa:
             inconsistent.
         """
         header = read_test_file(file, dir=dir)
-        self.assertObservationInfo(header, check_wcs=check_wcs, wcs_params=wcs_params, **kwargs)
+        self.assertObservationInfo(header, filename=file, check_wcs=check_wcs,
+                                   wcs_params=wcs_params, **kwargs)
 
-    def assertObservationInfo(self, header, check_wcs=True, wcs_params=None, **kwargs):  # noqa: N802
+    def assertObservationInfo(self, header, filename=None, check_wcs=True,  # noqa: N802
```

Comment:
I'm being dense, but what's the N802 infraction here?

Reply:
It's a naming complaint. This package tries to use underscores but in this particular method it's matching the non-standard convention of unittest.TestCase.